app/services/trials.js

```diff
@@ -86,6 +86,8 @@ function _processData(trials) {
 
         // Computed values here
         daysAfterCompletion: today.diff(trial.completion_date, 'days'),
+        daysBeforeCompletion: _cleanDate(trial.completion_date)
+          .diff(today, 'days'),
```

Comment:
I don't know exactly what this is doing, but I'm wondering why you're cleaning `trial.completion_date` on `daysBeforeCompletion` but not on `daysAfterCompletion`.